keras2onnx/_builtin.py

```diff
@@ -19,14 +20,19 @@ class TYPES:
     Any = 'Any'
     All = 'All'
     Cast = 'Cast'
+    StridedSlice = 'StridedSlice'
 
     # converter internal types:
     TD_Reshape = '_reshape_timedistributed'
 
 
+def _is_tensor_const(tensor):
```

Comment:
Don't reply this test because in the future the converter probably retrieve the value from these node which could be const folding.